frontend/packages/kubevirt-plugin/src/components/modals/boot-order-modal/boot-order-modal.tsx

```diff
@@ -35,6 +46,8 @@ const BootOrderModalComponent = ({
   );
   const [showUpdatedAlert, setUpdatedAlert] = React.useState<boolean>(false);
   const [showPatchError, setPatchError] = React.useState<boolean>(false);
+  const vm = asVM(vmLikeEntity);
+  const { vmi } = React.useContext(VMDashboardContext);
```

Comment:
IMO we should not used a context which is not part of this render tree. This one is used by dashboard tab, right?

Reply:
+1